src/SDKs/Monitor/Management.Monitor/Properties/AssemblyInfo.cs

```diff
@@ -3,16 +3,15 @@
 
 using System.Reflection;
 using System.Resources;
-using System.Runtime.InteropServices;
 
-[assembly: AssemblyTitle("Microsoft Azure Monitor Library")]
-[assembly: AssemblyDescription("Provides Microsoft Azure Monitor operations.")]
+// [assembly: AssemblyTitle("Microsoft Azure Monitor Library")]
```

Comment:
@shriku11 please uncomment these lines.
Also any reason you are commenting assembly meta data.

Reply:
This is fixed and added the commented line back. The local build was giving error and was suggesting to remove this.